commit a8596ca31e82cef6a47a4b0174e57ffa5ef92de1
Author: 6by9 <6by9@users.noreply.github.com>
Date:   Tue Dec 11 15:18:02 2018 +0000

    staging: bcm2835-camera: Check the error for REPEAT_SEQ_HEADER (#2782)
    
    When handling for V4L2_CID_MPEG_VIDEO_REPEAT_SEQ_HEADER was added
    the firmware would reject the setting if H264 hadn't already been
    selected. This was fixed in the firmware at that point, but to
    enable backwards compatibility the returned error was ignored.
    
    That was Dec 2013, so the chances of having a firmware that still
    has that issue is so close to zero that the workaround can be
    removed.
    
    Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>

diff --git a/drivers/staging/vc04_services/bcm2835-camera/controls.c b/drivers/staging/vc04_services/bcm2835-camera/controls.c
index 17b6960fc151..e7c3c68e40f6 100644
--- a/drivers/staging/vc04_services/bcm2835-camera/controls.c
+++ b/drivers/staging/vc04_services/bcm2835-camera/controls.c
@@ -1100,7 +1100,7 @@ static const struct bm2835_mmal_v4l2_ctrl v4l2_ctrls[V4L2_CTRL_COUNT] = {
 		0, 1, NULL,
 		MMAL_PARAMETER_VIDEO_ENCODE_INLINE_HEADER,
 		&ctrl_set_video_encode_param_output,
-		true	/* Errors ignored as requires latest firmware to work */
+		false
 	},
 	{
 		V4L2_CID_MPEG_VIDEO_H264_PROFILE,
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1396_a8596ca31e82cef6a47a4b0174e57ffa5ef92de1.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1396_a8596ca31e82cef6a47a4b0174e57ffa5ef92de1.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
